docs/api/taxons.rst

```diff
@@ -931,3 +931,62 @@ Exemplary Response
 .. code-block:: text
 
     STATUS: 204 No Content
+
+Set position of product in a Taxon
+----------------------------------
+
+The products in Sylius can by group by taxon, therefore every product has an relation between itself and selected taxon.
+What is more, every product can have specific position in taxon to which it is belongs to. To put products in specific order
+you will need to call the ``/api/v1/taxons/taxon_code/products`` endpoint wih the ``PUT`` method.
+
+Definition
+^^^^^^^^^^
+.. code-block:: text
+
+    PUT /api/v1/taxons/{taxonCode}/products
+
++---------------+----------------+-----------------------------------------------------------------+
+| Parameter     | Parameter type | Description                                                     |
++===============+================+=================================================================+
+| Authorization | header         | Token received during authentication                            |
++---------------+----------------+-----------------------------------------------------------------+
+| code          | url attribute  | Code of the taxon in which the order of product will be changed |
++---------------+----------------+-----------------------------------------------------------------+
+
+Example
+^^^^^^^
+
+To change order of products with code ``7f260b98-aad8-3932-b628-1ac135d47767`` and ``c7127357-db24-3931-adb8-e80fc44e1507`` in taxon with code ``womens_t_shirts`` use the below method:
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/taxons/womens_t_shirts/products \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json" \
+        -X PUT
+        --data '
+            {
+                "positionsData": [
+                    {
+                        "product_code": "7f260b98-aad8-3932-b628-1ac135d47767",
+                        "position": 3
+                    },
+                    {
+                        "product_code": "c7127357-db24-3931-adb8-e80fc44e1507",
+                        "position": 0
+                    }
+                ]
+            }
+        '
+
+.. note::
+
+    Remember the *7f260b98-aad8-3932-b628-1ac135d47767* and *c7127357-db24-3931-adb8-e80fc44e1507* and *womens_t_shirts* are just exemplary codes and
+    there could be selected other. Check in the list of all product if you are not sure which codes should be used.
```

Comment:
`are just exemplary codes and you can change them for the ones you need. Check in the list of all products if you are not sure which codes should be used.`